tests/accel/upgrade/internal/url/url.go

```diff
@@ -0,0 +1,63 @@
+package url
```

Comment:
This package is avaliable in /tests/assisted/internal/url/url.go
Please move it to upper level ./tests/internal/url.go and re-use in your code.

Reply:
Done, thanks